src/main/java/org/jabref/gui/openoffice/StyleSelectDialogView.java

```diff
@@ -0,0 +1,138 @@
+    @FXML private TableView<StyleSelectItemViewModel> tvStyles;
+    @FXML private TableColumn<StyleSelectItemViewModel, String> colJournals;
+    @FXML private TableColumn<StyleSelectItemViewModel, String> colFile;
+    @FXML private TableColumn<StyleSelectItemViewModel, Boolean> colDeleteIcon;
+    @FXML private Button add;
+    @FXML private VBox vbox;
+    private final MenuItem edit = new MenuItem(Localization.lang("Edit"));
+    private final MenuItem reload = new MenuItem(Localization.lang("Reload"));
+    @Inject private PreferencesService preferencesService;
+
+    private StyleSelectDialogViewModel viewModel;
+    private final DialogService dialogService;
+    private final StyleLoader loader;
+    private PreviewPanel previewArticle;
+    private PreviewPanel previewBook;
+
+    public StyleSelectDialogView(DialogService dialogService, StyleLoader loader) {
+        this.dialogService = dialogService;
+        this.loader = loader;
+
+        ViewLoader.view(this)
+                  .load()
+                  .setAsDialogPane(this);
+
+        setResultConverter(button -> {
+            if (button == ButtonType.OK) {
+                viewModel.storePrefs();
+                return tvStyles.getSelectionModel().getSelectedItem().getStyle();
+            }
+            return null;
+
+        });
+        setTitle(Localization.lang("Style selection"));
+    }
+
+    @FXML
+    private void initialize() {
+
+        viewModel = new StyleSelectDialogViewModel(dialogService, loader, preferencesService);
+
+        previewArticle = new PreviewPanel(null, new BibDatabaseContext(), preferencesService.getKeyBindingRepository(), preferencesService.getPreviewPreferences(), dialogService, ExternalFileTypes.getInstance());
+        previewArticle.setEntry(TestEntry.getTestEntry());
+        vbox.getChildren().add(previewArticle);
+
+        previewBook = new PreviewPanel(null, new BibDatabaseContext(), preferencesService.getKeyBindingRepository(), preferencesService.getPreviewPreferences(), dialogService, ExternalFileTypes.getInstance());
+        previewBook.setEntry(TestEntry.getTestEntryBook());
+        vbox.getChildren().add(previewBook);
+
+        colName.setCellValueFactory(cellData -> cellData.getValue().nameProperty());
+        colJournals.setCellValueFactory(cellData -> cellData.getValue().journalsProperty());
+        colFile.setCellValueFactory(cellData -> cellData.getValue().fileProperty());
+        colDeleteIcon.setCellValueFactory(cellData -> cellData.getValue().internalStyleProperty());
+
+        new ValueTableCellFactory<StyleSelectItemViewModel, Boolean>().withGraphic(internalStyle -> {
+            if (!internalStyle) {
+                return IconTheme.JabRefIcons.DELETE_ENTRY.getGraphicNode();
+            }
+            return null;
+
+        }).withOnMouseClickedEvent(item -> {
+            return evt -> viewModel.deleteStyle();
+        }).withTooltip(item -> {
```

Comment:
no braces needed for one line expressions...

Reply:
Nope, it's not a single consumer, it's a `Function<T, EventHandler<? super MouseEvent>>`